content/using/operations/using-your-ship.md

```diff
@@ -182,6 +182,13 @@ Because Urbit networking is stateful we call this a _continuity breach_. Everyth
 
 When this happens, back up any files you'd like to save, shut down your urbit, and recreate it (as if you were starting for the first time).
 
+### Life and rift number
+
+You can check your ship's _life_ and _rift_ number by running the `+keys`
```

Comment:
`+keys` does not seem to work. I get this:

```
> +keys
%cx /~lodleb-ritrul/home/~2020.2.25..22.15.24..9a61/gen/keys/hoon
scry failed for
ford: %hood failed for /~lodleb-ritrul/home/0/gen/keys/hoon:
ford: %core on /~lodleb-ritrul/home/0/gen/keys/hoon failed:
```

Reply:
Works on my ship. And [it certainly exists](https://github.com/urbit/urbit/blob/master/pkg/arvo/gen/keys.hoon). Something is way messed up about your home desk!

(It's `+keys ~ship`, though. So `+keys our` for our life and rift.)